mslib/mscolab/app/__init__.py

```diff
@@ -34,7 +35,10 @@
 from flask import Flask, url_for
 from mslib.mscolab.conf import mscolab_settings
 from flask_sqlalchemy import SQLAlchemy
-from mslib.utils import prefix_route
+from mslib.utils import prefix_route, release_info
+
+
+logging.debug(release_info.check_for_new_release())
```

Comment:
A debug message seems to be the wrong log level to me. I think it should be at least info, maybe even a warning, or not a log message at all and instead a print.

Reply:
The server output of a productive system usually is not seen on a terminal. 
No output and only a warning in the log when an update is needed is the better solution for the servers.